docs/conf.py

```diff
@@ -268,6 +268,7 @@
 intersphinx_mapping['specreduce'] = ('https://specreduce.readthedocs.io/en/stable/', None)
 intersphinx_mapping['photutils'] = ('https://photutils.readthedocs.io/en/stable/', None)
 intersphinx_mapping['traitlets'] = ('https://traitlets.readthedocs.io/en/stable/', None)
+intersphinx_mapping['roman_datamodels'] = ('https://roman_datamodels.readthedocs.io/en/stable/', None)
```

Comment:
oh fun, the repo and python package name have underscore, but on pip/RTD its a hyphen?  Is there anywhere else that might also need updates (`PYTEST_HEADER_MODULES` maybe)?

```suggestion
intersphinx_mapping['roman_datamodels'] = ('https://roman-datamodels.readthedocs.io/en/stable/', None)
```

Reply:
Huh, good catch! I think I updated it everywhere else but missed this one. Why is RTD not complaining? We have nitpicky

https://github.com/spacetelescope/jdaviz/blob/6a2892c1c321547be7ea11756fadb086387a49b3/docs/conf.py#L236